platforms/quorum/configuration/roles/helm_component/templates/constellation.tpl

```diff
@@ -22,6 +22,12 @@ spec:
       constellation: quorumengineering/constellation:{{ network.config.tm_version }}
     node:
       name: {{ peer.name }}
+{% if network.config.genesis | default('', true) | trim != '' %}
+{% if network.config.consensus == 'raft' %}
```

Comment:
dont think the 2nd if is necessary if you dont use it in IBFT

Reply:
Second check is necessary because the peer_id is not required for IBFT.

Reply:
okay